packages/components/src/section/header.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { useContext } from 'react';
```

Comment:
I think we should be importing this from `@wordpress/element`

Reply:
Updated here: https://github.com/woocommerce/woocommerce-admin/pull/7034/commits/e44d0962157186804e9e720ec535e29920b25235